src/seedsigner/gui/screens/tools_screens.py

```diff
@@ -180,7 +180,7 @@ def __post_init__(self):
         # Specify the keys in the keyboard
         self.rows = 2
         self.cols = 2
-        self.key_height = GUIConstants.TOP_NAV_TITLE_FONT_SIZE + 2 + 2*GUIConstants.EDGE_PADDING
+        self.key_height = GUIConstants.get_top_nav_title_font_size() + 2 + 2*GUIConstants.EDGE_PADDING
```

Comment:
We're not totally rigorous about this everywhere, but ideally there shouldn't be any hard-coded pixel values (the +2 after the title font size).

Whenever possible, stick to scaling a GUI constant (e.g. `int(GUIConstants.COMPONENT_PADDING / 4)`)